### DISCUSSION TITLE
Is possible change thumbnail size of image?

### DISCUSSION THREAD

**Discussion by FabrizioInfante** [laravel/nova-issues#4348] [Q&A]
Hello everyone,
I'm trying to change size of my image field's thumbnails from 32 to 100. I don't find anything in documentation so I presume to use custom CSS but how?

**Comment by philipbaginski** [comment]
You mean on Index page?

**Comment by FabrizioInfante** [comment]
Yes exactly, in the resource index table

**Comment by Illizian** [comment]
For those of you landing here from Google, you can use the methods provided by `Laravel\Nova\Fields\PresentsImages` trait:

```php
public function indexWidth(int $width) {}
public function detailWidth(int $detailWidth) {}
```
it also exposes methods for adjusting the aspect ratio:
```php
public function rounded() {}
public function squared() {}
public function aspect(string $aspect) {} // I think this accepts tailwind classes
```
HTH
